NEWS.md

```diff
@@ -1,11 +1,38 @@
 MXNet Change Log
 ================
+## 0.12.0
+### New Features - Sparse Tensor Support
+  - Added comprehensive support for sparse matrices. See help on `mx.sym.sparse` and `mx.nd.sparse` for more info.
+  - Limited support for fancy indexing. x[idx_arr0, idx_arr1, ..., idx_arrn] is now supported. Full support coming soon in next release. Checkout master to get a preview.
```

Comment:
fancy indexing is for normal NDArray

Reply:
@eric-haibin-lin Can you please edit the line and let me know how you want to rephrase it?

Reply:
I am just saying that this doesn't belong to sparse tensor

Reply:
Moved to other features. Done.